src/app/components/Profile/Profile.tsx

```diff
@@ -0,0 +1,73 @@
+import { useEffect, useState } from 'react';
+import defaultAvatar from './defaultAvatar.svg';
+import classes from './Profile.module.css';
+
+type Profile = {
+  username: string | null;
+  displayName: string | null;
+  avatar: string | null;
+};
+
+function openDialog() {
```

Comment:
If you name it like overwolf named it, it's more specific and understandable what the function does: `openLoginDialog()`

Reply:
thanks, renamed